activity_calendar/templates/activity_calendar/fullcalendar.html

```diff
@@ -52,120 +52,132 @@ <h1>Activity Calendar</h1>
+                      width="16"
+                      class="icon-success"
+                      style="display:none"
+                      id="subscribe-done"
+                    />
+                    <img
+                      src="{% static 'octicons/info.svg' %}"
+                      alt="Check-icon"
+                      height="16"
+                      width="16"
+                      style="display:none"
+                      id="subscribe-info"
+                    />
+                  </div>
+                  <!-- Subscription -->
+                  <div class="col nospace align-self-center">
+                    <small>
+                      <p id="event-subscription"></p>
+                      <p id="event-participants-count"></p>
+                      <p id="event-subscription-closed"></p>
+                    </small>
+                  </div>
+                </div>
 
-            <div class="row">
-              <!-- Calendar Icon -->
-              <div class="col-xs-2">
-                <img
-                  src="{% static 'octicons/three-bars.svg' %}"
-                  alt="Description-icon"
-                  height="16"
-                  width="16"
-                />
-              </div>
-              <!-- Event Description -->
-              <div class="col align-self-center">
-                <div id="event-description" class="rendered-md"></div>
+                <div class="row">
+                  <!-- Calendar Icon -->
+                  <div class="col-xs-2">
+                    <img
+                      src="{% static 'octicons/three-bars.svg' %}"
+                      alt="Description-icon"
+                      height="16"
+                      width="16"
+                    />
+                  </div>
+                  <!-- Event Description -->
+                  <div class="col align-self-center">
+                    <div id="event-description" class="rendered-md"></div>
+                  </div>
+                </div>
               </div>
             </div>
+            <div class="modal-footer">
+              <a id='event-subscribe-url' class="btn btn-primary" href="">View Registration Slots</a>
+              <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
+            </div>
           </div>
         </div>
-        <div class="modal-footer">
-          <a id='event-subscribe-url' class="btn btn-primary" href="">View Registration Slots</a>
-          <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
-        </div>
-      </div>
     </div>
-  </div>
+
+    <div class="mt-3">
+        <h5>Add this to your agenda</h5>
+        <p>You can import the Knights agenda into your own agenda. To do this follow the instructions on adding an
+        agenda from url and use the following url:</p>
+        <p class="">
+            https://app.kotkt.nl{% url "activity_calendar:icalendar" %}
```

Comment:
Hardcoded URLs are bound to break at some point. Perhaps look into the `build_absolute_uri` template tag (in `core`); it _almost_ works for this case.